draft-kazuho-httpbis-priority.md

```diff
@@ -233,18 +233,78 @@ set to `3` and the progressive parameter set to `1`.
 
-# Merging Client- and Server-Driven Parameters {#merging}
-
-It is not always the case that the client has the best understanding of how the
-HTTP responses deserve to be prioritized.  For example, use of an HTML document
-might depend heavily on one of the inline images.  Existence of such
-dependencies is typically best known to the server.
-
-By using the "Priority" response header, a server can override the
-prioritization hints provided by the client.  When used, the parameters found
-in the response header field overrides those specified by the client.
-
-For example, when the client sends an HTTP request with
+# Prioritization Process and Lifecycle
+
+HTTP/2 alludes to a prioritization process and does not specify many details
+about it. This might go some way to explain the different perspectives about
+prioritization that arise from different operators and use cases. The design
+also mixes concerns about prioritization between requests, streams and
+connections.
+
+HTTP/2's frame-based prioritization places an emphasis on the client's role in
+determining the priority of a response. {{?RFC7540}} Section 5.3 describes
+one key characteristic:
+
+> Explicitly setting the priority for a stream is input to a
+  prioritization process.  It does not guarantee any particular
+  processing or transmission order for the stream relative to any other
+  stream.  An endpoint cannot force a peer to process concurrent
+  streams in a particular order using priority.  Expressing priority is
+  therefore only a suggestion.
+
+In practice, this approach is quite one-sided and puts a burden on the endpoint
+with the most limited amount of information. A client processing HTML and
+generating requests generally has only a URL and a type (determined by an HTML
+element or other mechanism) on which to make a prioritization prediction.
+
+A server typically has more information about the resource than the client,
+including the context around it (such as linked representations). It uses local
+knowledge, with input from the client, to generate a single selected
+representation ({{?RFC7231}}, Section 3) and respond. During response
+generation, the server can possibly determine that the client's predicted
+prioritization was incorrect (perhaps sub-optimal) for how the selected
+representation will be used. An HTTP/2 server might use this information,
+together with other inputs, to send the response at a different effective
+priority than the client suggested. However, it has no explicit means on which
+to present such a finding to the client.
+
+The frame-based prioritization process weakens further when intermediaries are
+considered because only the client is permitted to provide an explicit signal.
+
+## A Collaborative Model
+
+A collaborative model is presented that attempts to abstract priority and
+collaboration from concerns about HTTP version and active connections.
+
+Resources have an initial priority that begins with a neutral value. Requests
```

Comment:
To to it seems that it is from this line that the text actually talks about "Lifecycle."

I like the approach of talking about the lifecycle and how the request and response header field are being merged. To rephrase, I'm in favor of keeping the lines below this paragraph at the current position.